lib/classes/plugin-manager.js

```diff
@@ -255,7 +254,6 @@ class PluginManager {
   }
 
   resolveEnterprisePlugin() {
-    if (config.getConfig().enterpriseDisabled) return null;
```

Comment:
Is this a potential breaking change or we consider it a cleanup of something that was never really made to be official?

Reply:
I treat it as something that was never official. There's even no command that would set that  (user will have to manually manipulate the config)

Reply:
Thanks for explanation 👍